torchdata/datapipes/iter/__init__.py

```diff
@@ -71,6 +71,7 @@
     DropperIterDataPipe as Dropper,
     FlatMapperIterDataPipe as FlatMapper,
     FlattenIterDataPipe as Flattener,
+    ShuffledFlatMapperIterDataPipe as ShuffledFlatMapper,
```

Comment:
Could you please also add ShuffledFlatMapper to `__all__` (line 138) and `docs/source/torchdata.datapipes.iter.rst`?

Reply:
done https://github.com/PierreGtch/TorchData/commit/fb1735436debb96a750fbd841798204de247f81c